### ISSUE TITLE
[BUG]  Error 1. Retrying in five minutes.

### ISSUE THREAD

**Issue by munbart** [Chia-Network/chia-blockchain#5879]
**My computer configuration is:**

- OS Windows 10
- AMD RYZEN 7 5800X (8-core 4.3GHz)
- 64 GB RAM (3600MHz)
- SSD M.2 4TB A-Data XPG SX8100 (100% free space)
- 5 16TB HHDs (90% free space each)

I have run in Chia app 5 paralel plottings:

1. plot k=32, with 3800MB, 4threads, temp my ssd, then hhd_1
2. plot k=32, with 3700MB, 4threads, temp my ssd, then hhd_2
3. plot k=32, with 3600MB, 2threads, temp my hhd_3, then hhd_3
4. plot k=32, with 3600MB, 2threads, temp my hhd_4, then hhd_4
5. plot k=32, with 3600MB, 2threads, temp my hhd_5, then hhd_5

So, my RAM usage 31%, 20/64gb, CPU ~45%.
After 4h of plotting, 
**1st plot 32% with errors,** 
2nd 83% in progress (OK), 
3-5 = 32% in progress (OK), 

I can not understand why I can't plotting in paralel on my 4TB ssd two ore more plots?
This issue reproduce it in chia v1.1.4, v1.1.5, v.1.1.6

**Comment by lvpcguru** [comment]
What is the error on the 1st plot?

**Comment by munbart** [comment]
@lvpcguru
Only read 0 of 262140 bytes at offset 25427580 from X:\\plot-k32-2021-05-23-17-36-95b604ca12162901c3f5b3db3d9c540f6c0095bd45a363f10d30769fc9258e55.plot.p2.t3.sort_bucket_034.tmp with length 336588710. Error 1. Retrying in five minutes.

**Comment by munbart** [comment]
When it happens, ssd disk become unavailable

**Comment by lvpcguru** [comment]
X is your m.2 drive right? and that is actually installed on the m.2 interface? or do you run one of those usb adapters? 

And your 100% sure that drive has space left on it? Thats the error I get when I overflow the disk

**Comment by lvpcguru** [comment]
How did you get it fixed?

**Comment by munbart** [comment]
@lvpcguru I have been reinstalled m2 ssd, thought maybe not fully connected and it become works fine. But not long, just one day and today again the same problem(
Also yesterday has an error of copying completed plot to hdd but this is another story

**Comment by munbart** [comment]
@lvpcguru Can be a problem that I run plotting on HDDs ?
Also I have added one more plot to ssd and it didn't start more than 4h

**Comment by munbart** [comment]
![chia_plot_stats](https://user-images.githubusercontent.com/5167602/119685352-f65d4900-be4d-11eb-909f-bdea027b14aa.png)

**Comment by munbart** [comment]
![chia_plot_stats](https://user-images.githubusercontent.com/5167602/119685525-1b51bc00-be4e-11eb-82f3-f12aeb960c7a.png)

**Comment by ogbevireo** [comment]
I have the same error 1 issue. 
I have had for about 3 days leading to my having to close my gui and delete the temporary plotting directory before starting to plot again. 

My computer configuration is:

Chia v.1.1.6
OS Windows 10
Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz
12 GB RAM
500GB HDD OS & plotting drive (454GB free space when I started the plot)
1TB HDD Final drive
I am running a single plot at a time with one queued:
**Plot size is: 32
Buffer size is: 7460MiB
Using 128 buckets
Using 2 threads of stripe size 65536**

Plot is currently at 90% but the log shows that the error began while  compressing table 7 in phase 3/4.

Kindly see attached screenshots.

Thank you.

![Screenshot (2)](https://user-images.githubusercontent.com/65548295/119772173-ae264100-beb6-11eb-9bd4-3a50d61d6db8.png)
![Screenshot (3)](https://user-images.githubusercontent.com/65548295/119772418-06f5d980-beb7-11eb-9648-5dfa3abffeb1.png)

**Comment by lvpcguru** [comment]
> I have the same error 1 issue.
> I have had for about 3 days leading to my having to close my gui and delete the temporary plotting directory before starting to plot again.
> 
> My computer configuration is:
> 
> Chia v.1.1.6
> OS Windows 10
> Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz
> 12 GB RAM
> 500GB HDD OS & plotting drive (454GB free space when I started the plot)
> 1TB HDD Final drive
> I am running a single plot at a time with one queued:
> **Plot size is: 32 Buffer size is: 7460MiB Using 128 buckets Using 2 threads of stripe size 65536**
> 
> Plot is currently at 90% but the log shows that the error began while compressing table 7 in phase 3/4.
> 
> Kindly see attached screenshots.
> 
> Thank you.
> 
> ![Screenshot (2)](https://user-images.githubusercontent.com/65548295/119772173-ae264100-beb6-11eb-9bd4-3a50d61d6db8.png)
> ![Screenshot (3)](https://user-images.githubusercontent.com/65548295/119772418-06f5d980-beb7-11eb-9648-5dfa3abffeb1.png)

Check your drives remaining space. That happens when the temp drive is full and can't accept writes.

**Comment by lvpcguru** [comment]
> @lvpcguru Can be a problem that I run plotting on HDDs ?
> Also I have added one more plot to ssd and it didn't start more than 4h

Herm, I am really not sure. The drive looks clear, that's not a lot of parallel runs on it either. Is it an old hardrive? How long has it been in use? SSDs do have a limited life and chia mining kills it I'd you don't watch your lifespan. Maybe check the drive in crystal drive or something? If it's new you can contact the manufacturer of the drive.

**Comment by lvpcguru** [comment]
> > @lvpcguru Can be a problem that I run plotting on HDDs ?
> > Also I have added one more plot to ssd and it didn't start more than 4h
> 
> Herm, I am really not sure. The drive looks clear, that's not a lot of parallel runs on it either. Is it an old hardrive? How long has it been in use? SSDs do have a limited life and chia mining kills it I'd you don't watch your lifespan. Maybe check the drive in crystal drive or something? If it's new you can contact the manufacturer of the drive.

And it looks like adata has a 5 year warranty on yours
